api/Notes/NotesRouter.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express');
+const router = require('express').Router();
+const Notes = require('./noteModel');
+const Users = require('../Users/userModel');
+
+//GET & POST requests w/o ID
+router
+    .route('/')
+        .get((req, res) => {
+            Note.find()
+                .then(note => {
+                    res.json(note)
+                })
+                .catch(error => {
+                    res.status(500).json({
+                        errorMessage: 'Your notes could not be retrieved'
+                    });
+                })
+        })
+
+        .post((req, res) => {
+            Note.create()
+                .then(note => {
+                    res.status(201).json(note)
+                })
+                .catch(error => {
+                    res.status(500).json({
+                        errorMessage: 'Your note was not successfully created'
+                    });
+                })
+        })
+//GET, DELETE, & PUT requests w/ID
+router
+    .route('/:id')
+        .get((req, res) => {
+            const { id } = req.params;
+
+            Note.findById(id)
+                .then(note => {
+                    res.json(note)
+                })
+                .catch(error => {
+                    res.status(500).json({
+                        errorMessage: `The note with id ${id} could not be retrieved`
+                })
+            })
+        })
+
+        .delete((req, res) => {
+            const { id } = req.params;
+
+            if(!Note.findById(id)) {
```

Comment:
I think you can omit this `if` statement. If there is no note with the given id, it will hit the `catch` when you call `Note.findById(id)`.